apps/client/src/router/index.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { describe, it, expect } from 'vitest'
+import { detectProjectslug } from './index.js'
+import { useProjectStore } from '@/stores/project.js'
+
+setActivePinia(createPinia())
+describe('test router functions: detectProjectslug', () => {
```

Comment:
Par convention les descriptions de tests doivent démarrer par `should [...]` (`it should make me happy to run this function`)